src/components/PostComment.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { postCommentToArticle } from "../../api";
+
+export default function PostComment({id, setComments}) {
+    const [postStatus, setPostStatus] = useState(null);
+
+    function handleCommentPost(e) {
+        e.preventDefault();
+
+        const comment = {
+            username: e.target[0].value,
+            body: e.target[1].value
+        }
+
+
+        postCommentToArticle(id, comment)
+        .then((response) => {
+            setComments((current) => {
+                return [response.comment, ...current] 
+            })
+        })
+        .catch((error) => {
+            console.log(error)
+        })
+    }
+
+    const testComment = {
```

Comment:
Please always remove unused code before committing. This would result in your code being rejected in a workplace.